app/assets/javascripts/govuk_publishing_components/components/step-by-step-nav.js

```diff
@@ -40,6 +40,15 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       this.$module.sessionStoreLink = this.$module.sessionStoreLink + '_' + this.$module.uniqueId
     }
 
+    this.$module.upChevronSvg = '<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg" class="gem-c-step-nav__chevron">' +
```

Comment:
More of a FYI comment than anything else, I've created and referenced these as image assets right now. 

There might be a cross-over point where we can share assets.

Reply:
I've chosen this just based on what I've observed in the rest of them gem but I'm not precious about which implementation we use at the moment. Both would work cross browser. I think down the line we should agree on an implementation and align accordingly. For me, performance and any potential accessibility implications are going to be the smoking guns here. So far I haven't given this a deep screen reader assessment ie: navigating into the show button and trying to navigate the accessibility tree, but I can try it out.

I personally think lets get this into the gem now and update as and when. Happy to chat about it though.